src/main/java/cuchaz/enigma/gui/stats/StatsGenerator.java

```diff
@@ -112,12 +112,20 @@ private void update(Map<String, Integer> counts, Entry<?> entry) {
     private boolean isObfuscated(Entry<?> entry) {
         String name = entry.getName();
 
-        if (obfuscationTestService != null && obfuscationTestService.testDeobfuscated(entry)) {
-            return false;
+        if (!obfuscationTestServices.isEmpty()) {
```

Comment:
The `if` statement here isn't necessary. If there are no services, the loop will do nothing.

Reply:
yeah bad code, should have done better